apps/wallet-mobile/src/features/Swap/SwapCard/SwapCard.tsx

```diff
@@ -0,0 +1,194 @@
+import {Ionicons} from '@expo/vector-icons'
```

Comment:
we have a chevron icon that accepts the direction, check if matches the current design